titan-server/src/accounts/excuses.rs

```diff
@@ -0,0 +1,45 @@
+use diesel::{prelude::*, MysqlConnection};
+
+use crate::models;
+use crate::schema;
+
+pub const EVENT_TYPES: &'static [&'static str] = &[
```

Comment:
Should this be moved to a separate table that can have event types expanded upon (or shrunk down) depending on the changes to event types in the clan?

This seems logically inconsistent with how we treat file entry types as a joinable table, but this is being treated as a static listing.

Reply:
That makes sense. I'll go ahead and add an `event_type` entity.